src/components/modals/DeleteOwnAccountModal.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+    <modal-no-teleport ref="baseModal" :action="action" @cancel="close(action.CANCEL)">
+        <template #header>
+            <p class="text-2xl font-semibold text-red-600">Delete Own Account</p>
+        </template>
+        <div v-if="busy" class="mx-auto">
+            <loading-spinner />
+        </div>
+        <div v-else class="flex flex-col">
+            <div class="my-2 w-full flex justify-center my-8">
+                <div class="fa-stack fa-2x">
+                    <i class="fas fa-circle fa-stack-2x text-red-200" />
+                    <i class="fas fa-exclamation fa-stack-1x text-red-600" />
+                </div>
+            </div>
+            <p>
+                <span class="font-bold text-red-600">Warning!</span>
+                You are about to delete your UC4 account. If you continue, all your saved data will be permanently deleted and you will no
+                longer be able to use the system.
```

Comment:
![image](https://user-images.githubusercontent.com/8998495/102237361-e3386b80-3ef4-11eb-871f-4b5d2d5981aa.png)
Not sure how specific we should get, but all chain data remains and is still readable and (given the secret used at enrollment, which is currently called governmentId but could be anything) is also still connectable to a given person

Reply:
Maybe chaning it to "all your saved personal data..."?

Reply:
I change the shade of the red components from 500 to 700. Now it is a little friendlier to the eye. I also adjusted the describing text.
![grafik](https://user-images.githubusercontent.com/63233799/102242535-70ca8a00-3efa-11eb-9cc0-cbb4404756a7.png)